sdk/cosmosdb/cosmos/review/cosmos.api.md

```diff
@@ -958,7 +958,9 @@ export class ItemResponse<T extends ItemDefinition> extends ResourceResponse<T &
 export class Items {
     constructor(container: Container, clientContext: ClientContext);
     batch(operations: OperationInput[], partitionKey?: string, options?: RequestOptions): Promise<Response_2<OperationResponse[]>>;
-    bulk(operations: OperationInput[], bulkOptions?: BulkOptions, options?: RequestOptions): Promise<OperationResponse[]>;
+    bulk(operations: OperationInput[], bulkOptions?: BulkOptions, options?: RequestOptions): Promise<OperationResponse[] & {
+        diagnostics: CosmosDiagnosticContext;
```

Comment:
Actually the challenge is with backward compatibility.
The diagnostics can't be included in each operation. since they belong to batch operations. And given the existing API definition is Promise<OperationResponse[]>. i.e an array instead of an object, the only option I see is to add the property to the array itself.

Reply:
No, What I mean instead of defining object this way
OperationResponse[] & {diagnostics: CosmosDiagnosticContext;}

why can't we create an interface for the same

Reply:
added.